examples/jsm/controls/ArcballControls.js

```diff
@@ -0,0 +1,2495 @@
+
+		this._press = new Hammer.Press( { event: 'press', pointers: 1, time: 0, enable: false } );
+		this._singlePan = new Hammer.Pan( { event: 'singlepan', pointers: 1, threshold: 0, direction: Hammer.DIRECTION_ALL } );
+		this._doublePan = new Hammer.Pan( { event: 'doublepan', pointers: 2, threshold: 0, direction: Hammer.DIRECTION_ALL } );
+		this._triplePan = new Hammer.Pan( { event: 'triplepan', pointers: 0, threshold: 0, direction: Hammer.DIRECTION_ALL } );
+		this._pinch = new Hammer.Pinch();
+		this._rotate = new Hammer.Rotate();
+		this._doubleTap = new Hammer.Tap( ( { event: 'doubletap', taps: 2, threshold: 6, posThreshold: 20 } ) );
+
+		this._manager.add( [ this._press, this._singlePan, this._doubleTap, this._doublePan, this._triplePan, this._pinch, this._rotate ] );
+
+		this._manager.get( 'singlepan' ).recognizeWith( 'doublepan' );
+		this._manager.get( 'singlepan' ).recognizeWith( 'pinch' );
+		this._manager.get( 'singlepan' ).recognizeWith( 'rotate' );
+		this._manager.get( 'singlepan' ).recognizeWith( 'triplepan' );
+
+		this._manager.get( 'doublepan' ).recognizeWith( 'pinch' );
+		this._manager.get( 'doublepan' ).recognizeWith( 'rotate' );
+		this._manager.get( 'doublepan' ).recognizeWith( 'triplepan' );
+
+		this._manager.get( 'pinch' ).recognizeWith( 'rotate' );
+
+		this._manager.get( 'triplepan' ).recognizeWith( 'rotate' );
+		this._manager.get( 'triplepan' ).recognizeWith( 'pinch' );
+
+		this._manager.get( 'doubletap' ).recognizeWith( 'press' );
+
+
+		this._manager.get( 'triplepan' ).requireFailure( 'singlepan' );
+		this._manager.get( 'triplepan' ).requireFailure( 'pinch' );
+		this._manager.get( 'triplepan' ).requireFailure( 'rotate' );
+		this._manager.get( 'triplepan' ).requireFailure( 'doublepan' );
+
+		this._manager.get( 'rotate' ).recognizeWith( 'pinch' );
+
+		this.setCamera(camera);
+
+		if ( this.scene != null ) {
+
+			this.scene.add( this._gizmos );
+		
+		}
+
+		this._manager.on( 'press', this.onPress );
+		this._manager.on( 'doubletap', this.onDoubleTap );
+
+		this._manager.on( 'singlepanstart', this.onSinglePanStart );
+		this._manager.on( 'singlepanmove', this.onSinglePanMove );
+		this._manager.on( 'singlepanend', this.onSinglePanEnd );
+
+		this._manager.on( 'doublepanstart', this.onDoublePanStart );
+		this._manager.on( 'doublepanmove', this.onDoublePanMove );
+		this._manager.on( 'doublepanend', this.onDoublePanEnd );
+
+		this._manager.on( 'pinchstart', this.onPinchStart );
+		this._manager.on( 'pinchmove', this.onPinchMove );
+		this._manager.on( 'pinchend', this.onPinchEnd );
+
+		this._manager.on( 'rotatestart', this.onRotateStart );
+		this._manager.on( 'rotatemove', this.onRotateMove );
+		this._manager.on( 'rotateend', this.onRotateEnd );
+
+		this._manager.on( 'triplepanstart', this.onTriplePanStart );
+		this._manager.on( 'triplepanmove', this.onTriplePanMove );
+		this._manager.on( 'triplepanend', this.onTriplePanEnd );
+
+		this.domElement.addEventListener( 'mousedown', this.onMouseDown );
```

Comment:
Do you mind using pointer events instead of mouse events? We are in the middle of a migration to pointer events and it would be good to avoid adding code which depends on `mousedown`, `mousemove` etc..